### PR TITLE
Update runtime to fd.o 18.08

### PR DESCRIPTION
**Pull request by FakeShemp**

### CODE REVIEW

File: com.bixense.PasswordCalculator.json [hunk 1]
```diff
@@ -43,27 +43,28 @@
...
 			"build-commands": [
-				"./waf configure install --release --prefix=/app"
+				"./waf configure install --release --prefix=/${FLATPAK_DEST}"
 			],
 			"sources": [
 				{
 					"type": "git",
-					"url": "https://github.com/jhasse/pwcalculator",
+					"tag": "v1.1",
+					"url": "https://github.com/jhasse/pwcalculator.git",
```
Comment (jhasse) [inline com.bixense.PasswordCalculator.json:67]: Why add `.git`?

File: com.bixense.PasswordCalculator.json [hunk 2]
```diff
@@ -43,27 +43,28 @@
...
+				"cp -r boost ${FLATPAK_DEST}/include/boost"
 			],
 			"cleanup": ["*"]
 		},
 		{
 			"name": "PasswordCalculator",
 			"buildsystem": "simple",
 			"build-commands": [
-				"./waf configure install --release --prefix=/app"
+				"./waf configure install --release --prefix=/${FLATPAK_DEST}"
```
Comment (jhasse) [inline com.bixense.PasswordCalculator.json:61]: Doesn't `${FLATPAK_DEST}` already include the slash?

File: com.bixense.PasswordCalculator.json [hunk 3]
```diff
@@ -18,18 +18,18 @@
 				"--with-gtk=3", "--disable-sdltest", "--disable-webkit", "--disable-webview",
 				"--disable-webviewwebkit"
 			],
-			"cxxflags": [
-				"-std=c++11"
-			],
+			"build-options" : {
+        "cxxflags": "-std=c++11"
```
Comment (jhasse) [inline com.bixense.PasswordCalculator.json:22]: Use tabs instead of spaces here.

**Review by jhasse — COMMENTED** [review]
Thanks! I'll update the json in my repo accordingly.

**Comment by barthalion** [comment]
Opened new pull request at https://github.com/flathub/com.bixense.PasswordCalculator/pull/3 as the author removed their branch.

**Pull request closed without merging**